pymongo/uri_parser.py

```diff
@@ -618,6 +621,28 @@ def _parse_kms_tls_options(kms_tls_options: Optional[Mapping[str, Any]]) -> dict
     return contexts
 
 
+def _detect_external_db(entity: str) -> bool:
```

Comment:
I know it's not required, but would you mind adding a short docstring for this? Why do we have a fixed set of external db's? Will this keep expanding?

Reply:
The current set of external DBs is specified in the ticket, but I expect that list to expand with time.